mesheryctl/internal/cli/root/app/testdata/offboard.output.golden

```diff
@@ -0,0 +1 @@
+Post "http://localhost:9081/api/experimental/application": no responder found
```

Comment:
This should not be here!!!

Reply:
@piyushsingariya  added some changes in offborad test, but I need help on this one.